gcp/pd-backed-nfs-server/versions.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    google   = "4.7.0"
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = "1.13.1"
+    }
+  }
```

Comment:
We probably shouldn't hard-pin the versions, especially `google`, that means that anybody using this module in his TF must conform to this exact version

Reply:
https://github.com/env0/k8s-modules/pull/3/commits/d95a1621c1a66d02ea9c072f7554464d2c8eb045